tasks/build.js

```diff
@@ -29,34 +60,18 @@ function run(task, opts) {
 	return new Promise(function(resolve, reject) {
 		if (opts.watch) {
 			console.log("Watching " + getGlob(task) + " and will trigger " + task);
-			gulp.watch.call(gulp, getGlob(task), [task])
-				.on('end', resolve)
-				.on('error', reject);
+			gulp.watch(getGlob(task), [task]);
 		} else {
-			gulp.start.call(gulp, [task])
+			tasks[task]()
 				.on('end', resolve)
 				.on('error', reject);
 		}
 	});
 }
 
-gulp.task('build-sass', function() {
```

Comment:
the tasks still exists, I've just stored the function in a variable so it can be shared above in `run` (`gulp.start` isn't really meant to be used, apparently, and non of the events were being fired anyway)